Improve readme and docs (#124)
ome/omero_search_engine: docs/configuration/configuration_installation.rst

Reviewer: I can see where the problem is with the formatting. The formatting is ok on https://omero-search-engine.readthedocs.io/en/latest/configuration/configuration_installation.html. But we need to lead the user to read https://omero-search-engine.readthedocs.io/en/latest/configuration/configuration_installation.html. I think we should do it in the README file.

The ansible playbook :omero_search_engine:`management-searchengine.yml <deployment/ansible/management-searchengine.yml>` has been developed to deploy the apps:

* It will configure and run the search engine and Elasticsearch
* It will configure and create the required folders
* It will configure the three apps and run them
* There is a variables file :omero_search_engine:`searchengine_vars.yml <deployment/ansible/searchengine_vars.yml>` that the user needs to edit before running the playbook.
  The variable names are self-explanatory and should be customized to the host machine

> * To check that the apps have been installed and run, the user can use ``wget`` or ``curl`` to call:

Reviewer: ```suggestion
* To check that the apps have been installed and are running, use ``wget`` or ``curl`` to call:
```
